pkg/bindinfo/binding_match.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bindinfo
+
+import (
+	"sync"
+
+	"github.com/pingcap/tidb/pkg/parser/ast"
+)
+
+// ExtractTableName gets all table names from ast.Node.
+func ExtractTableName(in ast.Node) []*ast.Node {
```

Comment:
How about renaming it to `CollectTableNames` and renaming the structure below to `tableNameCollector` to make them aligned with each other?

Reply:
Done